src/app/src/main/kotlin/org/vaccineimpact/api/app/context/RequestBodySource.kt

```diff
@@ -5,31 +5,18 @@ import javax.servlet.MultipartConfigElement
 sealed class RequestBodySource
 {
     // Later, we should not return a string, but a stream or sequence of lines
-    abstract fun getBody(context: ActionContext): String
+    abstract fun getFile(context: ActionContext): String
```

Comment:
I feel like this should remain as `getBody`, as `getFile` only makes sense in the case of the `HTMLMultipart` source. I think it would be better to, instead of harcoding "file" as the part to get, make the part name a constructor parameter of `HTMLMultipart`.

Then you can use it like `source = HTMLMultipart("file")`.

Reply:
I think the opposite! `getBody` only makes sense in the `Simple` case...